src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs

```diff
@@ -0,0 +1,786 @@
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("Microsoft.NETCore.App", NuGetVersion.Parse("2.0.9"))));
+
+            public static ReferenceAssemblies NetCoreApp21 { get; }
+                = new ReferenceAssemblies("netcoreapp2.1")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("Microsoft.NETCore.App", NuGetVersion.Parse("2.1.13"))));
+        }
+
+        public static class NetStandard
+        {
+            public static ReferenceAssemblies NetStandard10 { get; }
+                = new ReferenceAssemblies("netstandard1.0")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard11 { get; }
+                = new ReferenceAssemblies("netstandard1.1")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard12 { get; }
+                = new ReferenceAssemblies("netstandard1.2")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard13 { get; }
+                = new ReferenceAssemblies("netstandard1.3")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard14 { get; }
+                = new ReferenceAssemblies("netstandard1.4")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard15 { get; }
+                = new ReferenceAssemblies("netstandard1.5")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard16 { get; }
+                = new ReferenceAssemblies("netstandard1.6")
+                .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("1.6.1"))));
+
+            public static ReferenceAssemblies NetStandard20 { get; }
+                = new ReferenceAssemblies(
+                    "netstandard2.0",
+                    new PackageIdentity(
+                        "NETStandard.Library",
+                        NuGetVersion.Parse("2.0.3")),
+                    @"build\netstandard2.0\ref")
+                .AddAssemblies(ImmutableArray.Create("netstandard"));
+
+            ////public static ReferenceAssemblies NetStandard21 { get; }
+            ////    = new ReferenceAssemblies("netstandard2.1")
+            ////    .AddPackages(ImmutableArray.Create(new PackageIdentity("NETStandard.Library", NuGetVersion.Parse("2.0.3"))));
```

Comment:
📝 I'm planning to file a follow-up issue to support netstandard2.1, netcoreapp3.0, and netcoreapp3.1.

Reply:
Filed #398, #399, #400.